### PR TITLE
Adding constants for Kibibyte, Mebibyte, Gibibyte, etc.

### PR DESCRIPTION
**Pull request by barpavel**
Adding constants for Kibibyte, Mebibyte, Gibibyte, Tebibyte & Pebibyte and using them everywhere in tests.

Signed-off-by: Pavel Bar <pbar@redhat.com>

### CODE REVIEW

File: tests/python_test.py [hunk 1]
```diff
@@ -256,9 +258,9 @@ def test_acquire_release_resource(tmpdir, sanlock_daemon, size, offset):
 
 @pytest.mark.parametrize("align, sector", [
     # Invalid alignment
-    (1024, sanlock.SECTOR_SIZE[0]),
+    (KiB, sanlock.SECTOR_SIZE[0]),
     # Invalid sector size
-    (sanlock.ALIGN_SIZE[0], 8192),
+    (sanlock.ALIGN_SIZE[0], 8*KiB),
```
Comment (nirs) [inline tests/python_test.py]: Missing space:

    8 * KiB

File: tests/units.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+"""
+Constants for file/disk sizes.
+"""
+
+KiB = 1024
+MiB = 1024**2
+GiB = 1024**3
+TiB = 1024**4
+PiB = 1024**5
+EiB = 1024**6
```
Comment (nirs) [inline tests/units.py]: I don't think we use PiB or EiB, and we will probably never use them.

File: tests/daemon_test.py [hunk 3]
```diff
@@ -78,26 +79,26 @@ def test_init_resource(tmpdir, sanlock_daemon):
 
 def test_format(tmpdir, sanlock_daemon):
     path = tmpdir.join("rindex")
-    size = 1024**2 * 3
+    size = MiB * 3
```
Comment (nirs) [inline tests/daemon_test.py]: While we fix this we should also change the order to:

    3 * MiB

Which is more natural.

**Review by nirs — CHANGES_REQUESTED** [review]
Thanks, looks much nicer now!

2 issues:
- always put the count before the size (e.g. 3 * MiB instead of MiB * 3)
- keep space around *

**Comment by barpavel** [comment]
1) Removed "EiB" constant, decided to leave "PiB" constant just in case we'll need it in some foreseeable future.
2) Changed the order when multiplying constant:
`MiB * 3 `--> `3 * MiB`
3) Added space around multiplication sign constant when multiplying constant:
`8*KiB` --> `8 * KiB`
Note: I decided to NOT add spaces in 2 places where this would not feel natural.
`def check_guard(path, size, guard=b"X", guard_size=4*KiB)`

**Review by nirs — COMMENTED** [review]
Looks like duplicate of #11.

Please take the typos commit out of this PR.

**Comment by barpavel** [comment]
Removed the typos commit.

**Review by nirs — APPROVED** [review]
Awesome, thanks!

**Pull request merged**